Documentation/network-requirements.md

```diff
@@ -24,5 +24,4 @@ The information below describes a minimum set of port allocations used by Kubern
 | TCP      | 4194        | Master & Worker Nodes          | The port of the localhost cAdvisor endpoint |
 | UDP      | 4789        | Master & Worker Nodes          | flannel overlay network - *vxlan backend* |
 | TCP      | 10250       | Master Nodes                   | Worker node Kubelet API for exec and logs.                                  |
-| TCP      | 10255       | Master & Worker Nodes          | Worker node read-only Kubelet API (Heapster).                                  |
```

Comment:
We should leave this network requirement in but label it 'optional'. otherwise this lgtm and tests are passing.

Reply:
Updated